content/docs/details/troubleshooting-the-catalog/index.md

```diff
@@ -31,8 +31,8 @@ the URL of a catalog info YAML file. To view details for a particular entity cli
 ### Checking the refresh schedule
 
 The `last refresh time` column above gives the time of the most recent update to the entity. The column `next refresh time`
-gives the time the catalog will next check for updates. It should be noted the last refresh time only updates when
-there are changes rather than on every check.
+gives the time the catalog will next check for updates. However updates should come through in real time also if using 
+Github. It should be noted the last refresh time only updates when there are changes rather than on every check.
```

Comment:
GitHub has two capital letters.

```suggestion
GitHub. It should be noted the last refresh time only updates when there are changes rather than on every check.
```